src/plugins/auto/plugin.cpp

```diff
@@ -416,6 +416,12 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
         }
         // reset the strDevices to support devices
         strDevices = "";
+        auto devicesWithPriority = GetValidDevice(supportDevices, networkPrecision);
```

Comment:
Need to exclude the case that MULTI_DEVICE_PRIORITIES is set and not all devices with "-" prefix.

Reply:
Was considered inside GetValidDevice